### PR TITLE
Added prototype for riboseq rust recipe

### PR DESCRIPTION
**Pull request by eggzilla**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by eggzilla** [comment]
Please do not merge, i have to inquire about the LICENSE!!!

**Comment by eggzilla** [comment]
Waiting for the repository owner to add the LICENSE file :-)

**Comment by eggzilla** [comment]
The author added the GPL-3 licence :-)
Please merge

File: recipes/riboseq-rust/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - argparse
+    - matplotlib
+    - numpy
+    - pysam
+
+test:
+  commands:
+    - rust_nucleotide -h 
+about:
+  home: https://www.ucc.ie/en/biochemistry/people/
```
Comment (mbargull) [inline recipes/riboseq-rust/meta.yaml]:
```yaml
  home: https://lapti.ucc.ie/rust/
```
would be more appropriate, I think.

**Comment by eggzilla** [comment]
Thx, I fixed the link!

File: recipes/riboseq-rust/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - matplotlib
+    - numpy
+    - pysam
+
+test:
+  commands:
+    - rust_nucleotide -h 
+about:
+  home: https://lapti.ucc.ie/rust/
+  license: GPL
```
Comment (mbargull) [inline recipes/riboseq-rust/meta.yaml]: Can you add the license file and clarify the license with
```yaml
  license: GPL-3.0
  license_file: COPYING
```

File: recipes/riboseq-rust/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  fn: RUST_{{ version }}.tar.gz
+  url: https://lapti.ucc.ie/rust/scripts/RUST_{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True # [not py27 or osx]
+
+requirements:
+  build:
```
Comment (mbargull) [inline recipes/riboseq-rust/meta.yaml]: Oh, and we actually don't need any `build` requirements 😄 as we only copy (+some `sed`/`chmod`ing).

Comment (eggzilla) [reply recipes/riboseq-rust/meta.yaml]: Thanks for pointing that out :-)
Removed the build dependencies

File: recipes/riboseq-rust/meta.yaml (hunk 2 above)
Comment (eggzilla) [reply recipes/riboseq-rust/meta.yaml]: Updated the the license information according to your review.
Thanks @mbargull !

**Pull request merged**